GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -227,6 +227,25 @@ public static ArgumentString ResetCmd(ResetMode mode, string commit = null, stri
             };
         }
 
+        /// <summary>
+        /// Push a local reference to a new commit
+        /// This is similar to "git branch --force "branch" "commit", except that uo get a warning if commits are lost
+        /// </summary>
+        /// <param name="repoPath">Full path to the repo</param>
+        /// <param name="gitRef">The branch to move</param>
+        /// <param name="targetId">The commit to move to</param>
+        /// <param name="force">Push the reference also if commits are lost</param>
+        /// <returns>The Git command to execute</returns>
+        public static ArgumentString PushLocalCmd(string repoPath, string gitRef, ObjectId targetId, bool? force = false)
+        {
+            return new GitArgumentBuilder("push")
+            {
+                string.Format("file://{0}", repoPath),
```

Comment:
in this case, string interpolation is more readable...

Reply:
this and next line can be combined?

Reply:
I think Philippe mainly means:
```suggestion
                $"file://{repoPath}",
                $"{targetId}:{gitRef}",
```